sourcecode/hub/config/mail.php

```diff
@@ -53,6 +53,9 @@
 
         'ses' => [
             'transport' => 'ses',
```

Comment:
I'm struggling to find proper documentation for this, but could you set the transport to be `sesv2`, as per [this article](https://medium.com/@ahmedesa/how-to-use-amazon-ses-v2-in-laravel-benefits-and-implementation-guide-c28b8dc8b78d)?